ion-schema/tests/ion_schema_tests_runner.rs

```diff
@@ -88,14 +88,24 @@ const SKIP_LIST: &[&str] = &[
     case::nullable_text_type("ion-schema-tests/ion_types/$text.isl"),
     case::nullable_timestamp_type("ion-schema-tests/ion_types/$timestamp.isl")
 )]
-fn validation_tests(path: &str) {
-    print!("{}...", path);
+fn validation_tests(path_string: &str) {
+    // The #[test_resources] macro provides file paths that are relative to the cargo workspace root
+    // but when this test is run, it seems the current directory is the "ion-schema/" directory, so
+    // we need to adjust the path to ensure that they are relative to the correct location.
+    let mut path_buf = PathBuf::from_str(TEST_ROOT_DIR).unwrap();
+    if path_string.starts_with("ion-schema-tests") {
+        path_buf.pop();
+    }
```

Comment:
The `if` here suggests that `path_string` will sometimes _not_ start with `ion-schema-tests`. Is that the case, or can we simply call `path_buf.pop()` no matter what? The working directory of a call to `cargo test` is wherever the `Cargo.toml` file for that crate lives, so I think it should always be the same unless we reorganize the directory structure again.